cpp/src_prims/selection/columnWiseSort.cuh

```diff
@@ -175,7 +175,7 @@ template <typename InType, typename OutType>
 void sortColumnsPerRow(const InType *in, OutType *out, int n_rows,
                        int n_columns, bool &bAllocWorkspace, void *workspacePtr,
                        size_t &workspaceSize, cudaStream_t stream,
-                       InType *sortedKeys = nullptr) {
+                       InType *sortedKeys = nullptr, bool ascending = true) {
```

Comment:
There is a GTest in `columnSort.cu`. I'll add this param. I ended up not needing this change because the selection and sort are fused together w/ a min/max heap in a single kernel. I do think it's useful to have, though.